app/[lang]/Navbar.tsx

```diff
@@ -50,6 +50,15 @@ export default function Navbar({ lang }: { lang: Locale }) {
           `/${lang}/amils-story`,
         ],
       },
+      {
+        label: 'Media',
+        route: [
+          `/${lang}/news`,
+          `/${lang}/press-releases`,
+          `/${lang}/press`,
```

Comment:
are `/news` and `/press` routes needed in this PR? If not, I think it should be removed